modules/deploy/partials/kubernetes/guides/explore-topics-localhost.adoc

```diff
@@ -4,7 +4,7 @@ Redpanda Console is a developer-friendly web UI for managing and debugging your
 
 In this step, you use port-forwarding to access Redpanda Console on your local network.
 
-TIP: Because you're using the Community Edition of Redpanda Console, you should not expose Redpanda Console outside your local network. The Community Edition of Redpanda Console does not provide authentication, and it connects to the Redpanda cluster as superuser. To use the Enterprise Edition, you need a license key, see xref:get-started:licenses.adoc[Redpanda Licensing].
+TIP: Because you're using the Community Edition of Redpanda Console, you should not expose Redpanda Console outside your local network. The Community Edition of Redpanda Console does not provide authentication, and it connects to the Redpanda cluster as superuser. To use the Enterprise Edition, you need a license key, see xref:get-started:licensing/index.adoc[Redpanda Licensing].
```

Comment:
```suggestion
TIP: Because you're using the Community Edition of Redpanda Console, you should not expose Redpanda Console outside your local network. The Community Edition of Redpanda Console does not provide authentication, and it connects to the Redpanda cluster as superuser. To use the Enterprise Edition, you need a license key. See xref:get-started:licensing/index.adoc[Redpanda Licensing].
```